### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

**Review by xansec — CHANGES_REQUESTED** [review]
Does this fuzz radixdlt-scrypto, or just sbor? Please make the requested changes.

**Comment by evanrichter** [comment]
This implementation of sbor serialization is done by the radixdlt-scrypto project. It's not a vendored dependency written by someone else. You can see they maintain each of these sub-crates in tandem even: https://github.com/radixdlt/radixdlt-scrypto/commit/a411dc087ddf11e80700603fcf83dcde5954c089

File: sbor/fuzz/fuzz_targets/roundtrip.rs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#![no_main]
```
Comment (xansec) [inline sbor/fuzz/fuzz_targets/roundtrip.rs:1]: As long as this crate is part of the upstream repo that's fine with me.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**